README.md

```
@@ -87,7 +87,7 @@ terraform/
 ## 🚀 Development Setup
  **Public documentation**: [https://api.cezzis.com/prd/cocktails/api-docs/v1/scalar/v1](https://api.cezzis.com/prd/cocktails/api-docs/v1/scalar/v1)
 1) Prerequisites
-   - .NET SDK 9.0
+  - .NET SDK 10.0
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)